presentation/Main.java

```diff
@@ -52,10 +66,36 @@ public static void main(String[] args) {
                     );
                     System.out.println();
                 }
+                case Menu.SHOW_AND_SORT_STUDENTS -> {
+                    System.out.println("1: Sort by Student Id");
+                    System.out.println("2: Sort by First Name");
+                    System.out.println("3: Sort by Last Name");
+                    System.out.print("Enter your choice: ");
+                    int choice = scanner.nextInt();
+                    System.out.printf("%-15s%-15s%-15s%n","Student Id","First Name","Last Name");
+                    switch (choice){
```

Comment:
To reduce the code in the presentation layer, let the service decide how to handle the sorting. You could pass the `choice` to `studentService.sortStudentsBy(choice);` afterward, just output the already sorted list of students.